docs/docs-new/pages/product/workspace/semantic-layer-sync.mdx

```diff
@@ -265,6 +265,45 @@ sending them, or use a service account with a different email address:
 
 <Screenshot src="https://ucarecdn.com/2e7ac466-73a5-4dd6-9c85-ed98d9df8979/" />
 
+### Tableau
```

Comment:
Very curious: why do we need PRs to both versions of docs? Do we still maintain the old one?

Reply:
Only doing this temporarily to ensure links are migrated + redirected correctly, need some time to work on a separate PR that removes the old docs completely